modules/bootcamp/detect_landing_pad.py

```
@@ -98,31 +95,42 @@ def run(self, image: np.ndarray) -> "tuple[list[bounding_box.BoundingBox], np.nd
         # * conf
         # * device
         # * verbose
-        predictions = ...
+
+        predictions = self.__model.predict(
+            source=image,
+            conf=0.7,
+            device=self.__DEVICE,
+            verbose=False,
+        )
 
         # Get the Result object
-        prediction = ...
+        prediction = predictions[0]
 
         # Plot the annotated image from the Result object
         # Include the confidence value
-        image_annotated = ...
+        image_annotated = prediction.plot(conf=True)
 
         # Get the xyxy boxes list from the Boxes object in the Result object
-        boxes_xyxy = ...
+        boxes_xyxy = prediction.boxes.xyxy if prediction.boxes else []
+        print("boxes_xyxy:", boxes_xyxy)
 
         # Detach the xyxy boxes to make a copy,
         # move the copy into CPU space,
         # and convert to a numpy array
-        boxes_cpu = ...
+        boxes_cpu = [box.cpu().numpy() for box in boxes_xyxy]
+        print("boxes_cpu:", boxes_cpu)
 
         # Loop over the boxes list and create a list of bounding boxes
         bounding_boxes = []
         # Hint: .shape gets the dimensions of the numpy array
-        # for i in range(0, ...):
-        #     # Create BoundingBox object and append to list
-        #     result, box = ...
+        for i in range(0, len(boxes_xyxy)):
+            # Create BoundingBox object and append to list
+            result, box = bounding_box.BoundingBox.create(boxes_cpu[i])
+
+            if result and box:
+                bounding_boxes.append(box)
```

instead of skipping bad bounding boxes, we want to skip the whole image if any box creation fails, in data science, we don't wanna return partial data (in this case 1 image is 1 data point, and the boxes are parts of that data)